<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>NPM:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nama:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Kelas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tempat Lahir:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanggal Lahir:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Telepon:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Alamat:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="npm_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nama_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="kelas_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tempat_lahir_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="telepon_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="alamat_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="simpan_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Simpan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ubah</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hapus_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="batal_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Batal</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="tanggal_lahir_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
